src/api/app/assets/javascripts/webui/request_show_redesign/add_review.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
+  $('#add-review-dropdown-component .dropdown-item').on('click', function() {
```

Comment:
This could be moved where the `add-review-dropdown-component` element is defined, thus only having this JS code running on the view which contains this element.